src/libudev/libudev.c

```diff
@@ -173,7 +173,9 @@ _public_ struct udev *udev_new(void) {
                                 if (prio < 0)
                                         log_debug("/etc/udev/udev.conf:%u: invalid log level '%s', ignoring.", line_nr, val);
                                 else
+#ifdef UDEV_PROPAGATE_LOG
                                         log_set_max_level(prio);
+#endif
```

Comment:
Hm, but the code contains https://github.com/systemd/systemd/blob/master/src/libudev/libudev.c#L87
>  * Create udev library context. This reads the udev configuration file, and fills in the default values.

I'm not sure what to do about it :-)